src/java/org/apache/cassandra/config/Config.java

```diff
@@ -257,6 +257,9 @@
     public volatile int counter_cache_save_period = 7200;
     public volatile int counter_cache_keys_to_save = Integer.MAX_VALUE;
 
+    public long blacklisted_partitions_cache_size_warn_threshold_in_mb;
+    public int blacklisted_partitions_cache_refresh_period_in_sec = 600; //10 minutes
```

Comment:
**Idea:** Do we need to expose this? If we can expose a nodetool and JMX interfaces, this extra config can be avoided. If anyone is interested in refreshing the cache quickly after adding new blacklisted partitions, they can use these interfaces. I am just trying to see if we can reduce the number of configs and keep it simple.

Reply:
i've already added a nodetool command to force refresh a cache incase someone adds a new blacklisted partition to be cached. I am not sure of the likelyhood of someone wanting to change the frequency of cache refresh at run time. I see your point of reducing the config changes, but I am honestly not sure if we should shy away from adding a config.